lib/razorpay/invoice.rb

```diff
@@ -20,5 +20,43 @@ def self.fetch(id)
     def self.all(options = {})
       request.all options
     end
+
+    def self.update(id, data = {})
+      request.patch id, data
+    end
+
+    def self.issue(id)
+      request.post "#{id}/issue"
+    end
+
+    def self.cancel(id)
+      request.post "#{id}/cancel"
+    end
+
+    def update(data = {})
+      self.class.update id, data
+    end
+
+    def issue
+      self.class.issue(id)
+    end
+
+    def cancel
+      self.class.cancel(id)
+    end
+
+    # no delete
   end
 end
+
+__END__
+
+
+    # Coming Soon
+    ##
+    def self.charge(id)
+      request.post "#{id}/charge"
```

Comment:
Not currently usable. Let's leave this out till it is.

Reply:
Removed :white_check_mark: